internal/pkg/print/debug.go

```diff
@@ -38,8 +38,7 @@ func BuildDebugStrFromMap(inputMap map[string]any) string {
 	}
 	sort.Strings(keys)
 
-	var builder strings.Builder
-	builder.WriteString("[")
+	result := "["
```

Comment:
In my opinion, you can make it even more readable like this:

```
var keyValues []string
for _, key := range keys {
		value := inputMap[key]
		if isEmpty(value) {
			continue
		}
		keyValues = append(keyValues,  fmt.Sprintf("%s: %v", key, value))
}

result := strings.Join(keyValues, ",")
return fmt.Sprintf("[%s]", result)
```

No `if...else` block and string concatenation needed